spec/support/factory_girl.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+end
+
+# Test::Unit
+class Test::Unit::TestCase
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.